app/jobs/application_job.rb

```diff
@@ -1,7 +1,4 @@
 class ApplicationJob < ActiveJob::Base
-  # Automatically retry jobs that encountered a deadlock
-  # retry_on ActiveRecord::Deadlocked
-
-  # Most jobs are safe to ignore if the underlying records are no longer available
-  # discard_on ActiveJob::DeserializationError
+  sidekiq_options retry: false
```

Comment:
Huh, this is interesting, how come it's retry false here and then we specify a retry strategy below?
Is it like a deny all... and then list exceptions to that rule below kind of thing?

Reply:
We're using sidekiq as the backing service for activejob, and they both have (independent) retry logic enabled by default